storio-common-annotations-processor/src/main/java/com/pushtorefresh/storio/common/annotations/processor/StorIOAnnotationsProcessor.java

```diff
@@ -92,38 +98,91 @@ private TypeElement validateAnnotatedClass(@NotNull final Element annotatedEleme
     /**
      * Checks that element annotated with {@link StorIOColumnMeta} satisfies all required conditions
      *
-     * @param annotatedField an annotated field
+     * @param annotatedElement an annotated field
      */
-    protected void validateAnnotatedField(@NotNull final Element annotatedField) {
-        // we expect here that annotatedElement is Field, annotation requires that via @Target
+    protected void validateAnnotatedField(@NotNull final Element annotatedElement) {
```

Comment:
I also was thinking about it but didn't come up with a decent name. `validateAnnotatedFieldOrMethod`? `validateAnnotatedWithColumnAnnotationElement`?